### PR TITLE
GITC-394: polygon tx validation

### PR DESCRIPTION
**Pull request by chibie**
##### Description

This PR adds backend functionality for validation of polygon transactions

##### Testing

Screenshot 2021-09-16 at 05 55 21

### CODE REVIEW

File: app/dashboard/utils.py [hunk 1]
```diff
@@ -298,8 +298,13 @@ def get_web3(network, sockets=False):
         web3.main.Web3: A web3 instance for the provided network.
 
     """
-    if network in ['mainnet', 'rinkeby', 'ropsten']:
-        if sockets:
+    if network in ['mainnet', 'rinkeby', 'ropsten', 'testnet']:
+        if network == 'mainnet' and is_polygon:
+            network = 'polygon-mainnet'
+        elif network == 'testnet':
```
Comment (zacheryschiller) [inline app/dashboard/utils.py:304]: Should this be `elif network == 'testnet' and is_polygon:` or is `testnet` only used for polygon anyway so it is redundant?

Comment (gcc20) [reply app/dashboard/utils.py:304]: yes it's only used for polygon

Comment (chibie) [reply app/dashboard/utils.py:304]: yes it's only used for polygon

**Review by zacheryschiller — COMMENTED** [review]
LGTM!

**Review by zacheryschiller — APPROVED** [review]

**Review by nomebahi — APPROVED** [review]

**Review by nomebahi — CHANGES_REQUESTED** [review]
fix it
great work and gr8 project I love it
0x4677379C040a8a05724ab99F3bc3ED1EbA95bF57

File: app/dashboard/utils.py [hunk 2]
```diff
@@ -284,7 +284,7 @@ def ipfs_cat_requests(key):
         return None, 500
 
 
-def get_web3(network, sockets=False):
+def get_web3(network, sockets=False, is_polygon=False):
```
Comment (thelostone-mc) [inline app/dashboard/utils.py:287]: would it make sense to network_name or something generic and the value could be `polygon`?

Comment (chibie) [reply app/dashboard/utils.py]: done

**Review by thelostone-mc — APPROVED** [review]

**Pull request merged**